src/main/java/frc/robot/Command/ClimbCommand/ClimbSetSpeed.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.Command.ClimbCommand;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Subsystem.Climb.ClimbSubsystem;
+
+public class ClimbSetSpeed extends Command {
+
+  ClimbSubsystem climb = ClimbSubsystem.getInstance();
+  Double speed;
+  /** Creates a new ClimbSetSpeed. */
+  public ClimbSetSpeed(double speed) {
+    addRequirements(climb);
+    this.speed = speed;
```

Comment:
אורי אוהב לעשות את this אני פחות.
נהוג לשנות את
 **private** double ### speed;
 **private** double ### _speed;
 ולהשאיא את הספיד האחר כמו שהוא וככה יש לך הבדל